projects/bb3/agents/utils.py

```diff
@@ -650,7 +650,9 @@ def compute_perplexities(
             start_label = [i for i, off in enumerate(text_off) if off <= prompt_len]
             assert len(start_label) > 0
             start_label = start_label[-1]
-            all_log_probs = result['choices'][0]['logprobs']['token_logprobs']
+            all_log_probs = [
+                lp for lp in result['choices'][0]['logprobs']['token_logprobs'] if lp
```

Comment:
could we change to `if lp is not None`, as `lp==0` could happen

Reply:
It shouldn't be an issue in practice given it comes from the softmax. But who know what we will be using in the future, so added it lol.